make/checkupdates.py

```
@@ -1,24 +1,23 @@
 
-from .pull import pull
+from . import (
+    REPO,
+    image_exists,
+    image_hash,
+    image_qualified_name,
+    in_system,
+    in_system_output,
+    is_root,
+)
 from .hash import hash
```

Ruff (0.15.9)

[error] Import `hash` is shadowing a Python builtin

(A004)